browser/src/main/resources/workspace-explorer/index.html

```diff
@@ -9,15 +9,17 @@
 <!DOCTYPE html>
 <html>
 <head>
-    <link rel="stylesheet" href="./graphiql.css" />
+    <link rel="stylesheet" href="./graphiql.css"/>
     <script src="https://cdnjs.cloudflare.com/ajax/libs/react/0.13.3/react.min.js"></script>
     <script src="https://cdnjs.cloudflare.com/ajax/libs/fetch/0.9.0/fetch.min.js"></script>
-    <script src="./graphiql.min.js"></script>
+    <script src="./graphiql.js"></script>
 </head>
 <body>
 Loading...
 <script>
-
+    function getQueryStringValue(key) {
```

Comment:
This gets the GraphQL query from the URL parameters.  Allows you to share queries by just sending an URL.  You get the url for this page and it opens up to this window with this query in the query pane.  In my version, in the correct workspace as well.  So when you hit a bug in a query, you could test it in this page and then send me the URL and it should work in the same system - assuming I'm running the same server config and data you have.